go/go16/Dockerfile

```diff
@@ -17,6 +17,12 @@ FROM golang:1.6
 # Install python3 for trampoline.
 RUN apt-get update && apt-get install -y python3 && which python3
 
+# Install protoc
```

Comment:
Can we delete these old versions? Or separate PR for that?

Reply:
I'll do a follow up PR after this. Might be nice to have the proper git history just in case some day?!?